interface/frontend/detect_and_select.html

```diff
@@ -7,16 +7,109 @@
 {% endblock %}
 
 {% block content %}
+{{ block.super }}
+{% verbatim %}
```

Comment:
``{% verbatim %}`` always seems like a bad smell to me in Django. How come we need it?

Reply:
The Vue.js template syntax collides with the Django template syntax. Django would e.g. try to render following statements like `Candidate {{ index + 1 }}` or `{{ candidate.centroid.x }}` by itself, although the variables are provided by the Vue.js template language.

Reply:
> The Vue.js template syntax collides with the Django template syntax

Getcha. I was wondering if we could move the ``{% verbatim %}`` blocks to only surround the colliding javascript in that case? That would make it a bit clearer why, and implicitly so. Alternatively, we could move the JS to an external file. What do you think?

Reply:
Yes, I'd also prefer moving the tags so that they surround the "problematic" syntax as tightly as possible.